<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="1" total="5" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="test1" duration-ms="1084" started-at="2023-03-07T04:15:06Z" finished-at="2023-03-07T04:15:07Z">
    <groups>
    </groups>
    <test name="Test Report " duration-ms="1084" started-at="2023-03-07T04:15:06Z" finished-at="2023-03-07T04:15:07Z">
      <class name="com.qa.tests.GetTest01">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.tests.GetTest01@1e643faf]" name="setUp" is-config="true" duration-ms="16" started-at="2023-03-07T04:15:06Z" finished-at="2023-03-07T04:15:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="Test01()[pri:0, instance:com.qa.tests.GetTest01@1e643faf]" name="Test01" duration-ms="0" started-at="2023-03-07T04:15:06Z" finished-at="2023-03-07T04:15:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test01 -->
        <test-method status="PASS" signature="Test02()[pri:0, instance:com.qa.tests.GetTest01@1e643faf]" name="Test02" duration-ms="0" started-at="2023-03-07T04:15:06Z" finished-at="2023-03-07T04:15:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test02 -->
        <test-method status="PASS" signature="Test03()[pri:0, instance:com.qa.tests.GetTest01@1e643faf]" name="Test03" duration-ms="1" started-at="2023-03-07T04:15:06Z" finished-at="2023-03-07T04:15:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test03 -->
        <test-method status="FAIL" signature="Test04()[pri:0, instance:com.qa.tests.GetTest01@1e643faf]" name="Test04" duration-ms="1017" started-at="2023-03-07T04:15:06Z" finished-at="2023-03-07T04:15:07Z">
          <exception class="org.testng.internal.thread.ThreadTimeoutException">
            <message>
              <![CDATA[Method com.qa.tests.GetTest01.Test04() didn't finish within the time-out 1000]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.qa.tests.GetTest01.Test04() didn't finish within the time-out 1000
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.thread.ThreadTimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test04 -->
      </class> <!-- com.qa.tests.GetTest01 -->
    </test> <!-- Test Report  -->
  </suite> <!-- test1 -->
</testng-results>
